components/contact/ContactForm.jsx

```diff
@@ -1,14 +1,36 @@
 import React, { useState, useEffect, useCallback } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { toast } from 'react-toastify';
+import { makeStyles } from '@material-ui/core/styles';
 import Grid from '@mui/material/Grid';
 import Button from '@mui/material/Button';
 import TextField from '@mui/material/TextField';
+import InputLabel from '@mui/material/InputLabel';
 import CircularProgress from '@mui/material/CircularProgress';
 import { showFeedbackSuccess, setErrorModal } from '@reducers/ui';
 import { sendGitRequest } from '@reducers/data';
 import 'react-toastify/dist/ReactToastify.css';
 
+const useStyles = makeStyles(theme => ({
```

Comment:
avoid hard coding these css values where you can, specifically colors and font weights. prefer adding them to the global theme or reusing existing defined valued in the theme


https://github.com/hackforla/311-data/blob/main/theme/

Reply:
I went ahead and used the files from theme.